src/jobs/tag_version.yml

```diff
@@ -6,20 +6,34 @@ parameters:
     type: string
     default: "$(cat .version)"
   force:
-    description: "Use force to override versions"
+    description: "Use the force to override versions"
     type: boolean
     default: false
+  username:
+    description: "Username of the account used to push to Bitbucket/Github"
+    type: string
+    default: "codacycircleci"
+  password:
+    description: "Access token/app password of the account used to push to Bitbucket/Github"
+    type: string
+    default: "$GPR_AUTH_TOKEN"
 
 executor: machine
 
 steps:
   - attach_workspace:
       at: ~/workdir
-  - add_ssh_keys:
```

Comment:
keeping those lines will make this PR not breaking, I assume.

Reply:
It isn't technically breaking as I already added the environment variables to CircleCI. But I agree with marking the commit as breaking.

Reply:
How is this going to work for bitbucket repositories?

Reply:
Answer: Since circleci has different contexts per organization